scripts/publish_gh_release_notes.py

```diff
@@ -71,14 +72,19 @@ def main(argv):
         print("GH_RELEASE_NOTES_TOKEN not set", file=sys.stderr)
         return 1
 
+    slug = os.environ.get("TRAVIS_REPO_SLUG")
+    if not slug:
+        print("TRAVIS_REPO_SLUG not set", file=sys.stderr)
+        return 1
+
     rst_body = parse_changelog(tag_name)
     md_body = convert_rst_to_md(rst_body)
-    if not publish_github_release(token, tag_name, md_body):
+    if not publish_github_release(slug, token, tag_name, md_body):
         print("Could not publish release notes:", file=sys.stderr)
         print(md_body, file=sys.stderr)
         return 5
 
-    print(f"Release notes for {tag_name} published successfully")
+    print(f"Release notes for {tag_name} published successfully to {slug}")
```

Comment:
Might be cool to have a link to it there, assuming it's in the response somehow.
(the next time you debug/change it ;))

Reply:
Great idea, done. 👍